themes/startwords/layouts/_default/baseof.html

```diff
@@ -27,6 +27,11 @@
     <link rel="stylesheet" href="{{ $print.RelPermalink }}" media="print">
     {{ block "css" . }}{{ end }}
 
+    {{ $notes := resources.Get "js/notes.js" }}
+    {{/*     {{ $global := resources.Get "js/global.js" }} */}}
```

Comment:
is this line commented? not sure what it is/means

Reply:
Sorry - commented out based on the example from the hugo docs; I left it in because I thought we might have a main.js at some point. Should I remove it until we do?

Reply:
ah, got it - i guess there's no harm in leaving it since we should soon have a `main.js` with the logic for the main nav.